test/buildah-bud/apply-podman-deltas

```diff
@@ -252,8 +256,18 @@ skip_if_remote "different error messages between podman & podman-remote" \
 ###############################################################################
 # BEGIN temporary workarounds that must be reevaluated periodically
 
-skip "Test is completely broken, see buildah issue 4396" \
-     "bud-multiple-platform-values"
```

Comment:
I see no indication that https://github.com/containers/buildah/issues/4396 has been fixed or even looked at?

Reply:
NEVER MIND! I See it's skipped in buildah itself now.